test/types/fastify.test-d.ts

```diff
@@ -234,7 +233,6 @@ const ajvErrorObject: AjvErrorObject = {
   params: {},
   message: ''
 }
-expectAssignable<ValidationResult>(ajvErrorObject)
```

Comment:
I would add a guard here to run the assertion when the `params` is a string

Reply:
I'm sorry for late response. Did you mean to have a test like implemented in 9f6008f0353bfa9ea89e9f1e4334060a427574f2?